lib/exavier/server.ex

```diff
@@ -17,14 +14,14 @@ defmodule Exavier.Server do
   def handle_call(:xmen, _from, state) do
     # coverage is checked sequentially
     lines_to_mutate_by_module =
-      files()
-      |> Enum.reduce(%{}, fn {file, test_file}, acc ->
+      test_files()
+      |> Enum.reduce(%{}, fn test_file, acc ->
         module = Exavier.test_file_to_module(test_file)
-        lines_to_mutate = Exavier.Cover.lines_to_mutate(module, test_file)
+        {file, lines_to_mutate} = Exavier.Cover.lines_to_mutate(module, test_file)
 
         Map.put(acc, test_file, %{
-          file: file,
           module: module,
+          file: file,
```

Comment:
Before, I believe we were getting a binary (specifically, a string) as the `file` here. But from a quick test on a `iex` session, I can see it will now become a charlist. Doesn't this have any implications in further code?

Have you tried testing this with some sample mutations?

If CI passes, I'm confident this should be working, but I'd be more confident in merging as is (i.e., without `to_string`ing it) if we're triple sure. I can try and run a test on my end when I have more time, hopefully very soon!

Reply:
As far as I see. This path is used just in one [place](https://github.com/dnlserrano/exavier/blob/fe8020361db6295d9ec92cc71fb5170937c22833/lib/exavier.ex#L15). And [File.read!/1](https://hexdocs.pm/elixir/File.html#read!/1) is fine with charlists according to documentation. But I still can convert it to the string if you insist. It'll be still work with `string()` (which is `binary()`) as well

Reply:
No need to do it if it works fine. It was late and I wasn't sure I had grepped all usages correctly. Let's keep it as is, then. Thanks!